src/modules/auth/components/LogIn/LogInSection.tsx

```diff
@@ -0,0 +1,22 @@
+import { FormLogIn } from './FormLogIn.component';
+
+export function LogInSection(): JSX.Element {
+  return (
+    <div className="md:flex md:justify-center">
+      <div className="flex flex-col py-6 lg:w-1/2">
+        <FormLogIn />
+
+        <label className="label">
```

Comment:
It is better to set this element to be a `div` for example. Because `label` has specific behaviour... And it is used to better describe an `input` and help the user select it. 
eg. 
When you click on a label for some input, that input is going to be selected.